app/src/main/java/com/vietdung/mvparchitect/ui/MainActivity.java

```diff
@@ -0,0 +1,90 @@
+package com.vietdung.mvparchitect.ui;
+
+import android.app.SearchManager;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.SearchView;
+import android.view.Menu;
+import android.view.MenuItem;
+
+import com.vietdung.mvparchitect.data.repository.OnTaskListerner;
+import com.vietdung.mvparchitect.R;
+import com.vietdung.mvparchitect.data.model.User;
+import com.vietdung.mvparchitect.data.repository.SearchUserRepository;
+import com.vietdung.mvparchitect.data.source.remote.SearchUserRemoteDataSource;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MainActivity extends AppCompatActivity implements OnTaskListerner, SearchUserContract.View {
+
+    private RecyclerView mRecyclerView;
+    private List<User> mUsers;
+    private UserAdapter mUserAdapter;
+    private SearchView mSearchView = null;
+    private SearchUserContract.Presenter mSearchUserPresenter;
```

Comment:
Nếu chỉ có 1 presenter có thể để là
```suggestion
    private SearchUserContract.Presenter mPresenter;
```